web/docs/getting-started.md

```diff
@@ -33,12 +33,12 @@ We recommend using [nvm](https://github.com/nvm-sh/nvm) for managing your Node.j
 
   Then, install a version of node that you need, e.g.:
   ```shell-session
-  nvm install 16.15.0
+  nvm install 16.15.1
```

Comment:
This should also probably be `nvm install 16`. NVM will install the newest stable version it finds by default, but if the user already has a lower compatible version (e.g., `16.5.0`), we don't want to force them to install `16.15.11` too. This part of the docs can also be taken directly from the older revision I linked in the previous comment.

The reason we had `16.15.0` was to ensure NVM wouldn't install `16.15.1` (although in hindsight we should have probably specified the correct range).

P.S. Even if we did want to insist on specifying the newest stable version explicitly, we'd have to update this document every couple of weeks :smile:

Reply:
Yeah good points all around :) I will use `nvm install 16` and `nvm use 16` like before 👍🏻